frontend/public/components/resource-dropdown.tsx

```diff
@@ -71,10 +71,12 @@ const ResourceListDropdown_: React.SFC<ResourceListDropdownProps> = (props) => {
       }
 
       // Only show preferred version for resources in the same API group.
-      const preferred = (m: K8sKind) =>
-        preferredVersions.some(
-          (v) => v.groupVersion === apiVersionForReference(referenceForModel(m)),
-        );
+      const preferred = (m: K8sKind) => {
+        const grp = groupToVersionMap?.[m.apiGroup];
+        return grp
+          ? apiVersionForReference(referenceForModel(m)) === `${m.apiGroup}/${grp.preferredVersion}`
+          : false;
```

Comment:
Can't this just be...?

```suggestion
        return m.apiVersion === grp?.preferredVersion;
```

Reply:
The original logic was matching group version so I did not want to change it

Reply:
You're right we can just match the `preferred` version.